protocol/x/clob/keeper/clob_pair_test.go

```diff
@@ -1139,3 +1139,83 @@ func TestClobPairValidate(t *testing.T) {
 		})
 	}
 }
+
+func TestAcquireNextClobPairID(t *testing.T) {
+	memClob := memclob.NewMemClobPriceTimePriority(false)
+	mockIndexerEventManager := &mocks.IndexerEventManager{}
+	ks := keepertest.NewClobKeepersTestContext(t, memClob, &mocks.BankKeeper{}, mockIndexerEventManager)
```

Comment:
nit: Pass in a [no-op event manager](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/indexer/indexer_manager/indexer_manager_noop.go) instead so you don't have to add mocked method calls?